### PR TITLE
Todo cards

### PR DESCRIPTION
**Pull request by lh-rubz**

### CODE REVIEW

File: Task2/cards/cards.js [hunk 1]
```diff
@@ -0,0 +1,629 @@
...
+			});
+		})
+		.catch((error) => console.error("Error loading user data:", error));
+
+	const search = document.getElementById("searchBar");
+}
+
+// Function to add a user
+function addUser(userId) {
+	if (findUserById(userId)) {
```
Comment (MuathYHKefayah) [inline Task2/cards/cards.js]: no need for this

File: Task2/cards/cards.js [hunk 2]
```diff
@@ -0,0 +1,629 @@
...
+		return;
+	}
+
+	const newUser = new User(userId);
+	users.push(newUser);
+
+	// Create user card container
```
Comment (MuathYHKefayah) [inline Task2/cards/cards.js]: please move these to function

File: Task2/cards/cards.js [hunk 3]
```diff
@@ -0,0 +1,629 @@
...
+
+	updateTitle(newTitle) {
+		this.title = newTitle;
+	}
+
+	updateCompleted(newCompletedStatus) {
+		this.completed = newCompletedStatus;
+	}
+}
+function isCardIdTaken(cardId) {
```
Comment (MuathYHKefayah) [inline Task2/cards/cards.js]: if not needed please remove

File: Task2/cards/cards.js (hunk 1 above)
Comment (lh-rubz) [reply Task2/cards/cards.js]: done

File: Task2/cards/cards.js (hunk 2 above)
Comment (lh-rubz) [reply Task2/cards/cards.js]: done

File: Task2/cards/cards.js (hunk 3 above)
Comment (lh-rubz) [reply Task2/cards/cards.js]: done

**Review by MuathYHKefayah — APPROVED** [review]
Excellent Work!

**Pull request merged**